src/wasm/wasm-stack.cpp

```diff
@@ -2169,8 +2169,6 @@ void BinaryInstWriter::visitArraySet(ArraySet* curr) {
 
 void BinaryInstWriter::visitArrayLen(ArrayLen* curr) {
   o << int8_t(BinaryConsts::GCPrefix) << U32LEB(BinaryConsts::ArrayLen);
-  // Unused type index.
-  o << U32LEB(0);
```

Comment:
Shouldn't we emit the annotated version for now until V8 supports both? Or does it already?

Reply:
This is still emitting the annotated version, but the annotation is meaningless. V8 has supported both and ignored the annotation for a while now.